LogMonitor/src/LogMonitor/ConfigFileParser.cpp

```diff
@@ -660,7 +656,6 @@ void _PrintSettings(_Out_ LoggerSettings& Config)
             std::wprintf(L"\t\tDirectory: %ls\n", sourceFile->Directory.c_str());
             std::wprintf(L"\t\tFilter: %ls\n", sourceFile->Filter.c_str());
             std::wprintf(L"\t\tIncludeSubdirectories: %ls\n", sourceFile->IncludeSubdirectories ? L"true" : L"false");
-            std::wprintf(L"\t\tIncludeFileNames: %ls\n", sourceFile->IncludeFileNames ? L"true" : L"false");

```

Comment:
I'd missed that the json change removes the includeFileNames setting, I'm guessing it is now a default, right? @profnandaa @CharityKathure